BuildaUtils.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "BuildaUtils"
-  s.version      = "0.3.2"
+  s.version      = "0.4.2"
```

Comment:
@esttorhe yes! But I bumped it to 4.0 and then forgot to fix some stuff, so just continued bumping the version until it worked. Sorry a bit sloppy of me! 

Also, the SwiftSafe should be changed to not use my fork, It seams like my PR there has been merged: https://github.com/nodes-ios/SwiftSafe/pull/2